script/DeployRouter.s.sol

```diff
@@ -4,10 +4,22 @@ pragma solidity ^0.8.0;
 import {console2} from 'forge-std/console2.sol';
 import {DeployBase} from './DeployBase.s.sol';
 import {Router} from 'src/Router.sol';
+import {ICREATE3Factory} from 'create3-factory/ICREATE3Factory.sol';
 
 contract DeployRouter is DeployBase {
-    function _run(DeployParameters memory params) internal virtual override returns (address deployedAddress) {
-        deployedAddress = address(new Router(params.wrappedNative, params.pauser, params.feeCollector));
+    function _run(
+        DeployParameters memory params
+    ) internal virtual override isCREATE3FactoryAddressZero(params.create3Factory) returns (address deployedAddress) {
+        ICREATE3Factory factory = ICREATE3Factory(params.create3Factory);
+        bytes32 salt = keccak256('router');
+        bytes memory creationCode = abi.encodePacked(
+            type(Router).creationCode,
+            abi.encode(params.wrappedNative, msg.sender, params.pauser, params.feeCollector)
```

Comment:
I think we should have an `owner` in `local.json` to replace with `msg.sender`, similar to `pauser`.

Reply:
added